Assets/Drag.cs

```diff
@@ -35,11 +64,35 @@ private void StoppedMotion(ManipulationEventData arg0)
         {
             Destroy(gameObject); // delete the extra block
         }
-
+        else
+        {
+            transform.parent = console.transform;
+            gameObject.GetComponent<Drag>().enabled = false;
+            enableFlag = false;
+        }
+        if (clone_renderer != null)
+        {
+            foreach (var mat in clone_renderer.materials)
+            {
+                mat.SetFloat("_Outline", 0f);
+            }
+        }
+        if (renderer != null)
+        {
+            foreach (var mat in renderer.materials)
```

Comment:
see comment about elvis operator, this looks like duplicate code also, maybe this could be pulled out? Probably not worth it though